### DISCUSSION TITLE
How can EA change with respect to rotation?

### DISCUSSION THREAD

**Discussion by rhig2000** [wenbinyugroup/.github#38] [VABS Related Discussions] [answered]
Does it make sense that EA would change with respect to rotation in some situations?

For the simple example, EA does not change. EA = 8.2401×107 lbf, according to all of the following:
- Classical Stiffness Matrix (1,1)
- The extension stiffness EA
- Timoshenko Stiffness Matrix (1,1)
- Classical Stiffness Matrix at the Shear Center (1,1), and
- Computed using EA = 1/S′11 with S′11 = S11 − xT2 * S16 + xT3 * S15

However, I have other cross-sections where they differ slightly and not so slightly. For example:
- 5.3099302845E+07 per Classical Stiffness Matrix (1,1)
- 5.3099284723E+07 per The extension stiffness EA
- 5.3099302847E+07 per Timoshenko Stiffness Matrix (1,1)
- 5.3099302845E+07 per Classical Stiffness Matrix at the Shear Center (1,1), and
- 4.9537539669E+07 computed using EA = 1/S′11 with S′11 = S11 − xT2 * S16 + xT3 * S15

I'm not concerned about the slight differences within VABS output, but the last one computed using EA = 1/S′11 represents a significant difference (7%). Is this to be expected or must I have an error in my computation somewhere?

**Comment by rhig2000** [comment]
A response would be welcome

**Reply by wenbinyu-PU** [reply, marked as answer]
It is not surprising that the first four are close to each other. 

When you have couplings, you need to use the last option to define EA to account for the effects of coupling.  7% could be normal depending on the coupling. Consider the 3D stiffness matrix C11=E(1-nu)/(1+nu)/(1-2nu) and difference to E, for Poisson's ratio 0.3, the difference for an isotropic material will be around 34%.

**Reply by rhig2000** [reply]
Follow-up...
Tensile test measures extension stiffness of a material by pulling on a standardized sample. Since sideward deformation is not constrained for the test, isn't the Poisson effect "baked in"? Furthermore, our CAMRAD model isn't using shear coupling terms. So, for CAMRAD beam model, would it be more accurate to use one/all of the first 4 values?

**Comment by wenbinyu-PU** [comment]
In the inertial and elastic properties paper, I mentioned how to get the terms you needed. To discount shear-related deformation, one need to invert the corresponding 4x4 compliance matrix with respect to the shear center.
